src/mixins/common-mixin.ts

```diff
@@ -1,7 +1,9 @@
 import Vue from 'vue'
 import { Component } from 'vue-property-decorator'
+import { Getter } from 'pinia-class'
 import { RouteNames } from '@/enums/'
 import { ConfirmDialogType } from '@/interfaces/'
+import { useStore } from '@/store/store'
```

Comment:
Looks like we no longer use a Getter in this class, so this and line 3 can be deleted. Please and thanks!

Reply:
Fixed, thanks.